tests-functional/clients/status_backend.py

```diff
@@ -377,5 +377,5 @@ def input_connection_string_for_bootstrapping_another_device(self, connection_st
                 "clientConfig": {},
             },
         }
-        response = self.api_valid_request(method, data)
+        response = self.api_request(method, data)
```

Comment:
hmm other places (if any) where this method was used probably made the assumption that this request was gonna be a successful one. Are you sure this is a harmless change?

(btw no idea what it does, but isn't this a too generic spot for this method)